src/solc_0.5/Land/erc721/LandBaseTokenV3.sol

```diff
@@ -417,6 +419,7 @@ contract LandBaseTokenV3 is ERC721BaseTokenV2 {
         uint256 x,
         uint256 y
     ) internal {
+        require(size != 0, "size cannot be zero");
         require(x % size == 0 && y % size == 0, "Invalid coordinates");
         require(x <= GRID_SIZE - size && y <= GRID_SIZE - size, "Out of bounds");
         if (size == 3 || size == 6 || size == 12 || size == 24) {
```

Comment:
same here, this could be the require instad of `size != 0`

Reply:
after checking `_getQuadLayer` I dont think we need these changes at all